src/main/java/pages/SignUpPage.java

```diff
@@ -12,7 +12,7 @@
     private WebElement titleSingUpPage;
 
     @FindBy(xpath = "//input[@placeholder='Username']")
-    private WebElement inputUserName;
+    protected WebElement inputUserName;
```

Comment:
Same as above, I don't get this inheritance 🤔

Reply:
Because in this class is the webelement in common userName. :)